### PR TITLE
New package: sqlcipher-3.3.1

### PR DESCRIPTION
**Pull request by stpx**

### CODE REVIEW

File: srcpkgs/sqlcipher/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+revision=1
+build_style=gnu-configure
+configure_args="--enable-tempstore=yes"
+hostmakedepends="tcl"
+makedepends="libressl-devel sqlite-devel"
+short_desc="SQLite extension that provides transparent 256-bit AES encryption"
+maintainer="Steve Prybylski <sa.prybylx@gmail.com>"
+license="3-clause-BSD"
+homepage="https://www.zetetic.net/sqlcipher/"
+distfiles="https://github.com/sqlcipher/sqlcipher/"
```
Comment (dominikh) [inline srcpkgs/sqlcipher/template]: Duplicate distfiles line.

Comment (stpx) [inline srcpkgs/sqlcipher/template]: Whoops. Thanks

**Comment by xtraeme** [comment]
:+1: 

Looks like you've missed to add readline support...

**Comment by stpx** [comment]
Yeah.. I thought about that.. Will do.

**Comment by xtraeme** [comment]
:+1:

**Pull request merged**